skunkworks_crow/src/main/res/layout/activity_main.xml

```diff
@@ -21,37 +21,66 @@
             android:layout_centerHorizontal="true"
             android:layout_centerVertical="true"
             android:gravity="center"
-            android:textSize="22sp"
-            android:padding="10dp"/>
+            android:padding="10dp"
```

Comment:
avoid hard coding is better, you can replace the padding by `@dimen/default_padding_2.5x`.

Reply:
Until now we have not used `@dimen` so made like hardcoded or else I will make into that way only